openshift-api-for-data-protection-operator/base/subscription.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
```

Comment:
Please remove "labels" stanza, it will be added automatically upon creation.

Reply:
Np, will keep that in mind :)